anyway/flask_app.py

```diff
@@ -95,6 +95,29 @@
 assets = Environment()
 assets.init_app(app)
 assets_env = AssetsEnvironment(os.path.join(utilities._PROJECT_ROOT, "static"), "/static")
+assets.register('css_all', AssetsBundle(
```

Comment:
@atalyaalon see the changes here, please send to relevant people to review

this change allows to build the frontend assets ahead of time, then serve them from a separate nginx pod

(currently, they are built at run-time, which is not suitable for the new k8s infrastructure)

Reply:
@aviklai would be happy if you can go over this one (even though the pr is closed, we'll fix if we need)
@OriHoch I think that not all of static files are needed in the FE, but @aviklai can perhaps give a more insightful answer

Reply:
this replicates the existing behavior